blog/content/post/2020-03-24-trenchboot-nlnet-introduction.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Open Source DRTM with TrenchBoot for AMD processors. Introduction.
+abstract: This article starts an entire series of articles related to title
+          project. Today, you will find out why we have started such project and
+          who is supporting us. Also, we bring you closer to main concept and
+          goals.
+cover: /covers/nlnet-logo.png
+author: piotr.kleinschmidt
+layout: post
+published: false
+date: 2020-03-24
+archives: "2020"
+
+tags:
+  - trenchboot
+  - security
+  - open-source
+  - coreboot
+categories:
+  - Firmware
+  - Security
+
+---
+
+## Introduction
+
+Our company activity is to provide secure firmware and embedded systems to our
+customers. Our strategy is to use, contribute and spread open source solutions
+as much as possible. Factor, motivating to further work and showing the
+legitimacy of the adopted philosophy, is feedback in the form of satisfied
+customers, community and incoming foundations.
```

Comment:
customers and community. We also greatly appreciate the help from foundations which want to support our projects and ideas.
@pietrushnic I suggest such change. Is it ok?